databroker/src/broker.rs

```diff
@@ -579,7 +593,10 @@ impl Entry {
             self.actuator_target = actuator_target;
             changed.insert(Field::ActuatorTarget);
         }
-
+        if let Some(metadata_description) = update.description {
+            self.metadata.description = metadata_description;
+            // changed.insert(Field::ActuatorTarget);
+        }
```

Comment:
these changes at description are not mergeable we need to find another way to propagate the context to the client/provider.

Reply:
I was thinking if we could add this to the `types.proto` file

```
message MetadataMap {
  map<string, string> headers = 1;
}

message Metadata {
  optional MetadataMap otel_context = 18;  // to add this field alongside already existing fields in Metadata
}
```
By doing this, a new field is added to Metadata alongside description, while allowing the description to continue functioning as before _( I will revert all changes made to the description, ensuring it functions as it originally did )_.

MetadataMap could be used for otel context propagation and here is the output of `subscribe call` with the field `otel_context`: 

```
{
    "updates": [
        {
            "fields": [
                "FIELD_VALUE"
            ],
            "entry": {
                "path": "Vehicle.Speed",
                "value": {
                    "timestamp": {
                        "seconds": "1733672625",
                        "nanos": 127403578
                    },
                    "float": 22
                },
                "actuator_target": null,
                "metadata": {
                    "data_type": "DATA_TYPE_UNSPECIFIED",
                    "entry_type": "ENTRY_TYPE_UNSPECIFIED",
                    "value_restriction": null,
                    "description": "Odometer reading, total distance traveled during the lifetime of the vehicle.",
                    "unit": "km/h",
                    "otel_context": {
                        "headers": {
                            "tracestate": "",
                            "traceparent": "00-d9b7e6227856aefc7fe4ba0e142e2017-b6923a717115ffae-01"
                        }
                    }
                }
            }
        }
    ]
}
```

I have tested this change locally and it works as expected.

Please let me know if this approach is acceptable.

Reply:
no changes in proto as of now. Could you confirm that the tracing stuff is working with the context stuff removed?

Reply:
Yes, I have created a new pull request #119 containing just the tracing functionality. 
I have tested this change locally and it works fine.